core/src/main/java/com/linecorp/armeria/client/metric/AbstractMetricCollectingClient.java

```diff
@@ -47,7 +47,7 @@
 
     @Override
     public final O execute(ClientRequestContext ctx, I req) throws Exception {
-        RequestMetricSupport.setup(ctx, REQUEST_METRICS_SET, meterIdPrefixFunction, false);
+        RequestMetricSupport.setup(ctx, REQUEST_METRICS_SET, meterIdPrefixFunction, false, log -> false);
```

Comment:
Don't we need to customize the `successFunction` for `MetricCollectionClient`?

Reply:
Yes sorry I added them now